common/extensions/api/brave_shields.json

```diff
@@ -49,6 +49,24 @@
             "parameters": []
           }
         ]
+      },
+      {
+        "name": "dappAvailable",
+        "type": "function",
+        "description": "Called when website is dapp",
+        "parameters": [
+          {
+            "name": "tabID",
+            "type": "integer"
+          },
+          {
```

Comment:
this callback is unused and doesn't seem to have support in the c++ code

Reply:
This callback was removed.